tests/local/Dockerfile

```diff
@@ -0,0 +1,156 @@
+###
+### this Dockerfile is an example representing one variant of this image;
+### please see https://github.com/circleci-public/circleci-dockerfiles
+### for a complete list of Dockerfiles for each tag/variant of this image
+###
+
+FROM ubuntu:xenial
+
+# make Apt non-interactive
+RUN echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90harbor \
+  && echo 'DPkg::Options "--force-confnew";' >> /etc/apt/apt.conf.d/90harbor
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+  && mkdir -p /usr/share/man/man1 \
+  && apt-get install -y --no-install-recommends \
+    apt=1.2.32 apt-transport-https=1.2.32 build-essential=12.1ubuntu2 bzip2=1.0.6-8ubuntu0.2 \
+    curl=7.47.0-1ubuntu2.13 fonts-liberation=1.07.4-1 git=1:2.7.4-0ubuntu1.6 \
+    gnupg=1.4.20-1ubuntu3.3 gnupg2=2.1.11-6ubuntu2.1 gzip=1.6-4ubuntu1 \
+    iproute2=4.3.0-1ubuntu3.16.04.5 \
+    libappindicator3-1=12.10.1+16.04.20170215-0ubuntu1 libasound2=1.1.0-0ubuntu1 \
+    libatk-bridge2.0-0=2.18.1-2ubuntu1 libatk1.0-0=2.18.0-1 libatspi2.0-0=2.18.3-4ubuntu1 \
+    libcairo2=1.14.6-1 libgdk-pixbuf2.0-0=2.32.2-1ubuntu1.6 libgtk-3-0=3.18.9-1ubuntu3.3 \
+    libxss1=1:1.2.2-1 locales=2.23-0ubuntu11 mercurial=3.7.3-1ubuntu1.2 net-tools=1.60-26ubuntu1 \
+    netcat=1.10-41 openjdk-9-jdk-headless=9~b114-0ubuntu1 openssh-client=1:7.2p2-4ubuntu2.8 \
+    parallel=20161222-1~ubuntu0.16.04.1 postgresql=9.5+173ubuntu0.2 software-properties-common=0.96.20.9 \
+    sudo=1.8.16-0ubuntu1.7 tar=1.28-2.1ubuntu0.1 unzip=6.0-20ubuntu1 wget=1.17.1-1ubuntu1.5 \
+    xdg-utils=1.1.1-1ubuntu1.16.04.3 xvfb=2:1.18.4-0ubuntu0.8 zip=3.0-11 \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+# Set timezone to UTC by default
+RUN ln -sf /usr/share/zoneinfo/Etc/UTC /etc/localtime
+
+# Use unicode
+RUN locale-gen C.UTF-8 || true
+ENV LANG=C.UTF-8
+
+# install jq
+RUN JQ_URL="https://circle-downloads.s3.amazonaws.com/circleci-images/cache/linux-amd64/jq-latest" \
+  && curl --silent --show-error --location --fail --retry 3 --output /usr/bin/jq "$JQ_URL" \
+  && chmod +x /usr/bin/jq \
+  && jq --version
+
+# Docker & golang
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+RUN add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+RUN add-apt-repository ppa:longsleep/golang-backports
+
+RUN apt-get update && apt-get install -y --no-install-recommends containerd.io=1.2.6-3 \
+    docker-ce=5:19.03.2~3-0~ubuntu-xenial docker-ce-cli=5:19.03.2~3-0~ubuntu-xenial \
+    golang-go=2:1.13~1longsleep1+xenial \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+# docker compose
+RUN COMPOSE_URL="https://circle-downloads.s3.amazonaws.com/circleci-images/cache/linux-amd64/docker-compose-latest" \
```

Comment:
it's better install docker-compose from
https://github.com/docker/compose/releases/download/1.24.1/docker-compose-$(uname -s)-$(uname -m)